src/main.py

```diff
@@ -66,6 +66,22 @@ def updateResult(subName, result, score, timeLen, memUse, comment):
             "Admins have not yet upload the testcases. Go ahead and flame them.",
         )
         return
+    
+    for case in range(int(submission.testcase)):
+        if not Path(f"./source/{submission.problemId}/{case + 1}.in").is_file():
+            printFail("TESTCASE", f"Testcase {case + 1}.in is missing")
+            updateResult(
+                submission.id,
+                "Input missing",
+                0,
+                0,
+                0,
+                "Admins have not yet upload the testcases. Go ahead and flame them.",
+            )
+            return
+        
+        if not Path(f"./source/{submission.problemId}/{case + 1}.sol").is_file():
```

Comment:
if `.sol` is missing while have `.in`. How do user know?

Reply:
The .sol file usually comes paired with the .in file. So we can just check only .in files. More than that, some problems didn’t use the .sol file because the interactive script can generate solution text and  can evaluate without reading the .sol file.

Reply:
Like [Problem 706](https://api.otog.cf/problem/doc/706) which didn't use .sol files. It’s just using output from code and then evaluated.

Reply:
in case of user error. Ex. admin forget to generate `.sol` before upload and it must be get error. How it handle this case?